src/query_node.h

```diff
@@ -52,6 +52,12 @@ typedef enum {
   /* Vector */
   QN_VECTOR,
 
+  /* Verbatim */
+  QN_VERBATIM,
```

Comment:
Can you explain what this new QN is?

Reply:
Removed

I placed it when I thought we would have `'fooo'` as a token w/o need to escape special chars (but `'` and `\`)